docs/source/conf.py

```diff
@@ -17,7 +17,7 @@
 import sys
 
 # ensure tephi is discoverable by rtd build environment
-root = pathlib.Path(__file__).absolute().parent.parent.parent.parent
+root = pathlib.Path(__file__).absolute().parent.parent.parent
```

Comment:
😂

Not sure who wrote this originally, but for future reference Pathlib supports:

```suggestion
root = pathlib.Path(__file__).absolute().parents[2]
```

But I'm not going to hold up the review for this.